src/bot/commands/admin/activate-category.js

```diff
@@ -1,9 +1,45 @@
+import {query} from '../../../services/db';
 import {createCommand, createParser} from 'chatter';
+import {parseMatches, prepareMatchOutput, throwIfErrors} from '../../lib/matching';
 
 export default createCommand({
   name: 'activate category',
   description: 'Activate a new skills category.',
   usage: '<category name>',
-}, createParser(({args}) => {
-  return 'activate category command coming soon';
+}, createParser(({args}, {bot, token, getCommand}) => {
+  const search = args.join(' ');
+  if (!search) {
+    return false;
+  }
+  const output = [];
+  return query.categoryByName({token, search})
+    // parse matches
+    .then(results => parseMatches(results, search))
+    // return the matches and output
+    .then(prepareMatchOutput)
+    // handle any errors
+    .then(throwIfErrors)
+    // if a match exists, set the category as active
+    .then(results => {
+      const {exact} = results;
+      if (exact) {
+        return query.categoryIsActive({token, name: exact.name, isActive: true})
+          .then(() => {
+            output.push(`_You have successfully activated "${search}"._`);
+            return {output};
+          });
+      }
+      return results;
+    })
+    // return output for bot dialogue
+    .then(results => {
```

Comment:
ya, it's a bluebird sugar to do exactly what you're doing.

http://bluebirdjs.com/docs/api/get.html